CONTRIBUTING.md

```diff
@@ -0,0 +1,26 @@
+# How To Contribute
+
+## Installation
+
+* `git clone <repository-url>`
```

Comment:
Should this have a url of the current repository?

Reply:
I can make that change.